packages/extension/manifest/v3.json

```diff
@@ -13,11 +13,17 @@
     "default_title": "Alephium",
     "default_popup": "index.html"
   },
+  "host_permissions": [
+    "file://*/*",
+    "http://*/*",
+    "https://*/*"
+  ],
   "permissions": [
     "alarms",
     "tabs",
     "storage",
     "notifications",
+    "scripting",
```

Comment:
Are these new permissions required for OneKey as well?

Reply:
MetaMask requires this permission: https://github.com/MetaMask/metamask-extension/blob/ff635d238ce177c124e70b5f9e33a03d606e031a/app/manifest/v3/_base.json#L72, but OneKey doesn't. I tried using OneKey's approach, but it doesn't work.

Reply:
I just checked Rabby Wallet, and it also requires the `scripting` and `host_permissions` to solve this issue: https://github.com/RabbyHub/Rabby/blob/cc613b2acb8bb0e2872c8bba972b3fe85a75cb1e/src/manifest/mv3/manifest.json#L61

Reply:
Okay. Then let's use them